sklearn/multioutput.py

```diff
@@ -474,6 +474,40 @@ def predict_proba(self, X):
         results = [estimator.predict_proba(X) for estimator in self.estimators_]
         return results
```

Comment:
The docstring for the return type is wrong. It says `"array of shape (n_samples, n_classes), or a list of n_outputs such arrays if n_outputs > 1."` but this always returns a list.

If this estimator was meant to also handle the multilabel classification case (all targets are binary), then we could return this. This would be more consistent with other multiouput multiclass and multilabel classifiers such as `RandomForestsClassifier` and `KNeighborsClassifier`.

However this will break backward compat... I am not sure how to deal with this. This might be related to a discussion @glemaitre opened a while ago but I cannot find the issue.

Reply:
> However this will break backward compat... I am not sure how to deal with this. This might be related to a discussion @glemaitre opened a while ago but I cannot find the issue.

This classifier was probably not on my list of estimators because it is certainly not tested.
But as you mentioned, it seems that we changed the behaviour in 0.19 to always return a list.

I assume that we should only update the docstring and we would need a deprecation cycle if we decide to make all the `predict_proba` consistent.